packages/console/src/pages/ApiResources/index.module.scss

```diff
@@ -0,0 +1,23 @@
+@use '@/scss/underscore' as _;
+
+.headline {
+  display: flex;
+  justify-content: space-between;
+}
+
+.table {
+  margin-top: _.unit(6);
+  width: 100%;
+
+  tbody tr.clickable {
```

Comment:
Just wondering where comes the clickable className for tr elements?

Reply:
sorry. I miss it.
Added on the API Resource page.